client/components/domains/search-filters/dropdown-filters.jsx

```diff
@@ -149,6 +160,7 @@ export class DropdownFilters extends Component {
 			translate,
 		} = this.props;
 
+		const filterOnClose = abtest( 'domainSearchFilterOnClose' === 'enabled' );
```

Comment:
This one throws an error because `'domainSearchFilterOnClose' === 'enabled'` evaluates to false. I guess you meant to do `abtest( 'domainSearchFilterOnClose' ) === 'enabled'`? If yes then it works okay.

Reply:
Oh, good call. I can't believe I missed that...